src/org/omegat/core/data/RealProject.java

```diff
@@ -1029,6 +1029,12 @@ public void parseHeadFile(File file) throws Exception {
                         public void rebaseAndSave(File out) throws Exception {
                             mergeTMX(baseTMX, headTMX, commitDetails);
                             projectTMX.exportTMX(config, out, false, false, true);
+
+                            // it can be not loaded yet
+                            ProjectTMX newTMX = new ProjectTMX(config.getSourceLanguage(), config.getTargetLanguage(),
+                                    config.isSentenceSegmentingEnabled(),
+                                    new File(config.getProjectInternalDir(), OConsts.STATUS_EXTENSION), null);
+                            projectTMX.replaceContent(newTMX);
```

Comment:
Is it better to be called after `ProjectTMX#replaceContent`?
```
 projectTMX.exportTMX(config, out, false, false, true);
```

Reply:
current implementation export TMX files before rebase and commit. Is it intended behavior?